packages/web/src/components/search/DataSearch.js

```diff
@@ -929,6 +931,16 @@ class DataSearch extends Component {
 		const defaultSuggestions = isPopularSuggestionsEnabled
 			? [...this.normalizedRecentSearches, ...defaultPopularSuggestions]
 			: this.normalizedRecentSearches;
+		defaultSuggestions.map((suggestion) => {
```

Comment:
@anik-ghosh-au7 `_score` is not a good way to compare. It can have side effects later. We have different variables for recent and popular suggestions. You can add the key there.
![alt i](https://i.imgur.com/zciieqE.png)